### DISCUSSION TITLE
The "purchase" column no longer indicates the number of morphine already available on the submarine.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13826] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![2024-04-25_08-28-26](https://github.com/FakeFishGames/Barotrauma/assets/47295383/b4285e3b-201c-4658-8d08-e4c0542bc9f3)

![2024-04-25_08-30-49](https://github.com/FakeFishGames/Barotrauma/assets/47295383/2888616f-f346-4371-813b-9392f6a84969)

![2024-04-25_08-31-16](https://github.com/FakeFishGames/Barotrauma/assets/47295383/e9fa7cca-2501-4944-a369-f08e6a759c01)

![2024-04-25_08-31-53](https://github.com/FakeFishGames/Barotrauma/assets/47295383/fff60e92-4506-406c-b7e5-506ca92004fc)

### Reproduction steps

1. Captain must have "Traveling Tradesman" talent
2. Without filter string "owned" not seen
3. Try to use filter "morphine", then string "owned" overlapped

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I created [a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13831), we'll look into it!

**Comment by Regalis11** [comment]
Fixed in the latest hotfix
